api/v1/articles/index.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @defgroup api_v1_articles Article API requests
+ */
+
+/**
+ * @file api/v1/articles/index.php
```

Comment:
These wrappers are required for `PageHandler`s but not `ComponentHandler`s. It would be nice if we could avoid their use here.